### ISSUE TITLE
Follow followers' followers... & Location targeting

### ISSUE THREAD

**Issue by fombercom** [instabot-py/instabot.py#1005]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [x] I have searched for other issues with the same problem.

## Hosting
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [x] Feature Request
- [x] Technical Help
- [ ] Other

### Operating System? (include version)
MAC

## Python Version?
3

## Description

Basically.. It would be good to target the followers of the people that follow your page.. Hope that makes sense? Would work well for local networking, rather than international tags? 
[sorry if this is already part of the program]

&

It would be great if you could use the locations attached to instaposts to target followers.. e.g. I want to target people in Bristol, England specifically to like and follow.. 
[I think there was something for this posted somewhere but I couldn't located it]

**Comment by andrewsegas** [comment]
Here we have follow and like by location + user interface
https://github.com/andrewsegas/instaboost

**Comment by ChipWolf** [comment]
If this is still an issue, please open a new issue. Thanks!
